Contributions-to-existing-projects.md

```diff
@@ -38,20 +35,57 @@ The procedure described in the following is designed to ensure that the company
+> It is your responsibility to notify the Foundation when any change is required to the list of designated employees authorized to submit Contributions on behalf of the Corporation, or to the Corporation's Point of Contact with the Foundation.
+	
+Implementing this requirement is causing quite some organizational effort:
+* The signed CCLA has to made available inside the organization - This can be done via publishing the CCLA on the OSPOs website at a location which can be found easily be the employees (e.g. it might be usefull to have a "top level page" named CCLAs, this page then contains a list of "signed CCLAs", a list of "CCLAs under evaluation", a list of "denied CCLAs".)
+* All affiliates need to be informed and a procedure needs to be defined how the affiliates can nominate/ de-nominate contributors working for them. This becomes even more challenging in case an affiliate has no access to the intranet of the signing entity. In this case the signed CCLA or the information about the signed CCLA needs to be sent to the OSPOs of all affiliates, in case an affiliate has no OSPO set up, the information has to be routed to the function, which is in charge of sw development. All affiliates need to provide the names of nominated contributors or former contributors, who shall not be entitled anymore to do contributions to the OSPO of the signing entity, which then has to inform the Foundation/project about the change of the list of contributors. 
+* Publishing the list of contributors inside the organization and disclosing it to the Foundation/project might also require the approval of the data protection officers of the involved entities
+
+All this additional effort may hold organizations off to contribute small bugfixes or patches or even new features to the upstream OSS projects and puts them to risk of private forks and thus a lot of additional development effort in the long run. Thus the decision not to contribute needs to be taken very carefully. 
+
+A DCO in contrast to a CLA is a much more lightweight procedure. It was introduced to enhance the confidence that contributions to the Linux kernel are made "legally correct" by the contributors. The [DCO version 1.1](https://developercertificate.org/) is used by many OSS projects.
+
+The main difference of a DCO compared to a CLA is, that a DCO is not a contract, it is a kind of attest of the specific contributor that he/she is entitled to submit a concrete contribution. All the effort which has to be spend to get a CLA signed and maintained is not needed. The only tasks which have to be carried out are:
+* Evaluation of the DCO by the "Legal department"
+* Evaulation by the "IP department"
+* Evaluation by the specific contributor, whether it is acceptable for him/her
+Since the DCO version 1.1 is the "standard" the "Legal"- and "IP department" only have very little effort to spend.
```

Comment:
Maybe better: Projects have standardized on the DCO version 1.1, so the "Legal" ...

And adding a link to the DCO also would be good.